### PR TITLE
docs: update repositories.rst

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- update repositories.rst

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/installation/repositories.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 CodeIgniter Repositories
 ########################
 
+.. contents::
+    :local:
+    :depth: 2
+
+codeIgniter4 organization
```
Comment (paulbalandan) [inline user_guide_src/source/installation/repositories.rst:8]: Is the lowercased `c` intended?

Comment (paulbalandan) [reply user_guide_src/source/installation/repositories.rst:8]: Then I suppose the `i` should be, too.

**Review by MGatner — APPROVED** [review]
I agree with Paul, either all PascalCase or all lower. Rest looks good!

File: user_guide_src/source/installation/repositories.rst (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/installation/repositories.rst]: Oops, I overlooked that.

**Review by MGatner — APPROVED** [review]

**Pull request merged**